src/jobs/newsChecker/index.js

```diff
@@ -23,22 +23,19 @@ module.exports = async function(){
 
 const sendMail = async (sourceName) => {
     try{
-        const transporter = nodeMailer.createTransport({
-            service:'gmail',
-            auth:{
-                user: process.env.NOTIFY_EMAIL_ID,
-                pass:process.env.NOTIFY_EMAIL_PASSWORD
-            }
-        })
+        const mailgunApiKey = process.env.MAILGUN_API_KEY
+        const domain = process.env.MAILGUN_DOMAIN
+        const from = 'shiva.siristechnology@gmail.com'
```

Comment:
@ShivaAryal can you move this to environment variables? Please, do not hardcode constants.